src/components/AppVariantSelector.svelte

```diff
@@ -0,0 +1,25 @@
+<script>
+  import { createEventDispatcher } from "svelte";
+
+  export let variants = [];
+  export let selectedAppVariant = variants[0];
+  let selectedAppId;
+
+  const dispatch = createEventDispatcher();
+
+  const change = () => {
+    selectedAppVariant = variants.find((v) => v.app_id === selectedAppId);
+
+    dispatch("change");
+  };
+</script>
+
+<!-- svelte-ignore a11y-no-onchange -->
```

Comment:
I looked at this and it's interesting. The suggested change is to use `on:blur` instead so each keyboard press through the variants doesn't trigger an event. But that doesn't quite work so well since you actually have to de-focus the select box to trigger it. Anyway, just wondering if you looked at that also. I agree with silencing this warning.

Reply:
Yup! I investigated this and skipping seemed like the better solution here.